code/src/components/Header.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { Route, Link } from 'react-router-dom'
+
+import { ReactComponent as Mustache } from 'media/mustache.svg'
+
+import './Header.css'
+
+export const Header = () => {
+  return (
+    <header>
+      <h1 className='page-title'>Epic mustaches movies list!</h1>
+      <Route path='/movies'>
+      <Link to={"/"}>
+      <span className='mustache-icon'><Mustache /><p className='go-back'>Go back to movies</p></span>
```

Comment:
It's tricky business to nest anything inside `span` elements - they are inline, so a lot of styles of children might be messed up. Consider using `span` only in the final level (so, without any nesting below) and replace it over here with `div`